chicken-sloths/bangazon-site-sprint3
# data gen

# Description
- Add `faker` code to generate JSON files for all our model generation.
- Add `npm run data:gen` to regenerate the JSON files with new data.

## Problem to Solve
Generates JSON files with which to `bulkCreate` the incoming Sequelize models.

## Steps to Test Solution
1. `npm install`
1. `npm run data:gen`
1. Check that the JSON files change and adhere to the [ERD](https://github.com/chicken-sloths/bangazon-site-sprint3/wiki/Products-ERD).
